pkg/vere/io/http.c

```diff
@@ -459,6 +484,34 @@ _http_seq_link(u3_hcon* hon_u, u3_hreq* req_u)
   fig_u->seq_u = req_u;
 }
 
+/* _http_spin_unlink(): remove spin stack request from state
+*/
+static void
+_http_spin_unlink(u3_hreq* req_u)
+{
+  u3_hfig* fig_u = &req_u->hon_u->htp_u->htd_u->fig_u;
+  if ( 0 != req_u->pre_u ) {
+    req_u->pre_u->nex_u = req_u->nex_u;
+
+    if ( 0 != req_u->nex_u ) {
+      req_u->nex_u->pre_u = req_u->pre_u;
+    }
+  }
+  else {
+    fig_u->siq_u = req_u->nex_u;
+
+    if ( 0 != req_u->nex_u ) {
+      req_u->nex_u->pre_u = 0;
+    }
```

Comment:
should we have `else { uv_timer_stop(...); }` here?

Reply:
added